src/modules/partition/gui/ChoicePage.cpp

```diff
@@ -983,6 +984,51 @@ ChoicePage::updateActionChoicePreview( InstallChoice choice )
         m_previewAfterFrame->show();
         m_previewAfterLabel->show();
 
+
+        {
```

Comment:
These braces can be removed at last. (there for visibility)

Reply:
So this adds (duplicates) a whole chunk of code from the Erase/Replace case into Alongside as well. At this point I'd insist on refactoring it so that there's a function that returns a widget with bootloader-selection combo. That can keep the lambda's connected to the signals, and we can paste the widget into the layout for Erase or Alongside as needed.